src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        ColorSupplier supplier = new ColorSupplier();
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(supplier.getRandomColor());
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```
we should print balls, not colors